docs/resources/scenarios/task-reminders.md

```diff
@@ -0,0 +1,243 @@
+
+12. Next, the flow needs to process each Employee in the array returned by the script. Press **New step** and select **Post an Adaptive Card to a Teams user and wait for a response**.
+
+13. For the **Recipient** field, add **email** from the dynamic content (the selection will have the Excel logo by it). Adding **email** causes the flow step to be surrounded by an **Apply to each** block. That means the array will be iterated over by Power Automate.
+
+14. Sending an Adaptive Card requires the card's JSON to be provided as the **Message**. You can use the [Adaptive Card Designer](https://adaptivecards.io/designer/) to create custom cards. For this sample, use the following JSON.  
+
+    ```json
+    {
+      "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+      "type": "AdaptiveCard",
+      "version": "1.0",
+      "body": [
+        {
+          "type": "TextBlock",
+          "size": "Medium",
+          "weight": "Bolder",
+          "text": "Update your Status Report"
+        },
+        {
+          "type": "Image",
+          "altText": "",
+          "url": "https://i.imgur.com/f5RcuF3.png"
+        },
+        {
+          "type": "TextBlock",
+          "text": "This is a reminder to update your status report for this month's review. You can do so right here in this card, or by adding it directly to the spreadsheet.",
+          "wrap": true
+        },
+        {
+          "type": "Input.Text",
+          "placeholder": "My status report for this month is...",
+          "id": "response",
+          "isMultiline": true
+        }
+      ],
+      "actions": [
+        {
+          "type": "Action.Submit",
+          "title": "Submit",
+          "id": "submit"
+        },
+        {
+          "type": "Action.OpenUrl",
+          "title": "Open Spreadsheet",
+          "url": "https://microsoft.sharepoint.com/:x:/t/OfficePlatform/EZS8IHK8HV5FiQIp4IyiD-4BCZ0FR0r0oSkL4f3jQJBP4g?e=OQRrlZ",
+          "id": "open-spreadsheet"
+        }
+      ]
+    }
+    ```
+
+15. Fill out the remaining fields as follows:
+
+    - **Update message**: Thank you for submitting your status report. Your response has been successfully added to the spreadsheet.
+    - **Should update card**: Yes
+
+16. In the **Apply to each** block, following the **Post an Adaptive Card to a Teams user and wait for a response**, press **Add an action**. Select **Excel Online (Business)**. Under **Actions**, select **Run script (preview)**. Provide the following entries for the flow step:
+
+    - **Location**: OneDrive for Business
+    - **Document Library**: OneDrive
+    - **File**: task-reminders.xlsx
+    - **Script**: Save Status
+    - **senderEmail**: email *(dynamic content from Excel)*
+    - **statusReportResponse**: response *(dynamic content from Teams)*
+
+    ![The apply to each flow step.](../../images/scenario-task-reminders-last-flow-step.png)
```

Comment:
I don't think this is the right tooltip...?

Reply:
The image is of the "apply-to-each" block automatically added to the flow by Power Automate. I'm open to ideas for tooltip improvement, but the goal of the image is to highlight that both the card and second script are in that block.
![image](https://user-images.githubusercontent.com/38896772/84086425-ba12ad80-a99c-11ea-8703-d9286091d97d.png)